firstvoices/backend/resources/songs.py

```diff
@@ -0,0 +1,58 @@
+from import_export import fields
+from import_export.widgets import ForeignKeyWidget
+
+from backend.models import Lyric, Song
+from backend.models.constants import Visibility
+from backend.resources.base import BaseResource, SiteContentResource
+from backend.resources.utils.import_export_widgets import ChoicesWidget
+
+
+class SongResource(SiteContentResource):
```

Comment:
This resource could use the `RelatedMediaResourceMixin` in `backend/resources/base.py`